package.json

```diff
@@ -37,37 +37,35 @@
     "docs:publish": "cd hygen.io && yarn deploy"
   },
   "dependencies": {
-    "@types/node": "^17.0.19",
     "chalk": "^4.1.2",
     "change-case": "^3.1.0",
-    "debug": "^4.3.3",
+    "debug": "^4.4.0",
     "degit": "^2.8.4",
-    "ejs": "^3.1.6",
-    "enquirer": "^2.3.6",
-    "execa": "^5.0.0",
+    "ejs": "^3.1.10",
+    "enquirer": "^2.4.1",
+    "execa": "^5.1.1",
     "front-matter": "^4.0.2",
-    "fs-extra": "^10.0.0",
-    "ignore-walk": "^4.0.1",
-    "inflection": "^1.12.0",
-    "ora": "^5.0.0",
-    "yargs-parser": "^21.0.0"
+    "fs-extra": "^11.3.0",
+    "ignore-walk": "^7.0.0",
+    "inflection": "^1.13.4",
+    "ora": "^5.4.1",
```

Comment:
The last versions of the following packages are not compatible:
- enquirer
- execa
- inflection
- ora